new-integration-tests/src/test/resources/bash-scripts/createRepository.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+. /u01/oracle/wlserver/server/bin/setWLSEnv.sh
+
+echo "Check if the DB Service is ready to accept request "
+connectString=${1:-oracle-db.default.svc.cluster.local:1521/devpdb.k8s}
+schemaPrefix=${2:-domain1}
+rcuType=${3:-fmw}
+sysPassword=${4:-Oradoc_db1}
+
+echo "DB Connection String [$connectString], schemaPrefix [${schemaPrefix}] rcuType [${rcuType}]"
+
+max=100
+counter=0
+while [ $counter -le ${max} ]
+do
+ java utils.dbping ORACLE_THIN "sys as sysdba" ${sysPassword} ${connectString} > dbping.err 2>&1
+ [[ $? == 0 ]] && break;
+ ((counter++))
+ echo "[$counter/${max}] Retrying the DB Connection ..."
+ sleep 10
+done
+
+if [ $counter -gt ${max} ]; then
+ echo "[ERROR] Oracle DB Service is not ready after [${max}] iterations ..."
```

Comment:
If DB connection fails after all the retries, its helpful to see the actual error its failing for ...can you cat dbping.err here to print out the actual error

Reply:
fixed